framework/src/controller/application.js

```diff
@@ -111,28 +111,28 @@ module.exports = class Application {
 	/**
 	 * Register module with the application
 	 *
-	 * @param {Object} moduleSpec - Module specification
+	 * @param {Object} moduleKlass - Module specification
```

Comment:
Naming looks klassy but Why not using `moduleClass` :D ?

Reply:
Nothing special, but nothing wrong either. I saw this convention is widely used.